charts/handbrake/2.0.0/questions.yaml

```diff
@@ -0,0 +1,797 @@
+                  default: false
+                  hidden: true
+                  editable: false
+              - variable: datasetName
+                label: "Name"
+                schema:
+                  type: string
+                  default: "config"
+                  editable: false
+              - variable: mountPath
+                label: "Mount Path inside the container"
+                schema:
+                  type: path
+                  default: "/config"
+                  editable: false
+              - variable: hostPathEnabled
+                label: "config - Enable Host Path"
+                schema:
+                  type: boolean
+                  default: false
+                  show_subquestions_if: true
+                  subquestions:
+                    - variable: hostPath
+                      label: "config Host Path"
+                      description: "Host path to store config"
+                      schema:
+                        type: hostpath
+                        required: true
+          # storage
+        - variable: storage
+          label: "storage Volume"
+          schema:
+            type: dict
+            $ref:
+              - "normalize/ixVolume"
+            attrs:
+              - variable: enabled
+                label: "Enabled"
+                schema:
+                  type: boolean
+                  default: true
+                  required: true
+                  hidden: true
+                  editable: false
+              - variable: setPermissions
+                label: "Automatic Permissions"
+                description: "Automatically set permissions on install"
+                schema:
+                  type: boolean
+                  default: true
+                  hidden: false
+              - variable: emptyDir
+                label: "emptyDir"
+                schema:
+                  type: boolean
+                  default: false
+                  hidden: true
+                  editable: false
+              - variable: datasetName
+                label: "Name"
+                schema:
+                  type: string
+                  default: "storage"
+                  editable: false
+              - variable: mountPath
+                label: "Mount Path inside the container"
+                schema:
+                  type: path
+                  default: "/storage"
+                  editable: false
+              - variable: hostPathEnabled
+                label: "storage - Enable Host Path"
+                schema:
+                  type: boolean
+                  default: false
+                  show_subquestions_if: true
+                  subquestions:
+                    - variable: hostPath
+                      label: "storage Host Path"
+                      description: "Host path to store storage"
+                      schema:
+                        type: hostpath
+                        required: true
```

Comment:
```suggestion
          # storage ------------------------                        
        - variable: storage
          label: "storage dataset"
          schema:
            type: dict
            $ref:
              - "normalize/ixVolume"
            attrs:
              - variable: enabled
                label: "Enabled"
                schema:
                  type: boolean
                  default: true
                  required: true
                  hidden: true
                  editable: false
              - variable: setPermissions
                label: "Automatic Permissions"
                description: "Automatically set permissions on install"
                schema:
                  type: boolean
                  default: true
                  hidden: false
              - variable: emptyDir
                label: "emptyDir"
                schema:
                  type: boolean
                  default: false
                  hidden: true
                  editable: false
              - variable: datasetName
                label: "Dataset Name"
                schema:
                  type: string
                  default: "storage"
                  required: true
                  editable: false
              - variable: mountPath
                label: "Mount Path"
                description: "Path to mount inside the pod"
                schema:
                  type: path
                  required: true
                  default: "/storage"
                  editable: false
              - variable: hostPathEnabled
                label: "host Path Enabled"
                schema:
                  type: boolean
                  default: false
                  show_subquestions_if: true
                  subquestions:
                    - variable: hostPath
                      label: "Host Path"
                      schema:
                        type: hostpath
                        required: true
```
Standardize storage part2
(splitting because github dont like big suggestions)